### PR TITLE
Adding rx_ros to documentation index for distro melodic

### PR DESCRIPTION
**Pull request by henrik7264**
I'd like rx_ros to be indexed and documented on ros.org.

### CODE REVIEW

**Review by clalancette — CHANGES_REQUESTED** [review]
This is failing Travis CI because it isn't a valid ROS package.  That is, it is missing the required package.xml that defines it as a package.  Please see http://wiki.ros.org/ROS/Tutorials/CreatingPackage, http://wiki.ros.org/ROS/ReleasingAPackage, and http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease to setup your repository.

**Comment by sloretz** [comment]
@clalancette It looks like the repository is a catkin workspace with a ROS package inside https://github.com/rosin-project/rx_ros/blob/master/src/rxros/package.xml . The package is called `rxros`, but the workspace is called `rx_ros`. CI is complaining about the latter.

```
RuntimeError: melodic: missing package.xml file for package "rx_ros"
```

The setup is a little unusual; I'm not sure what it would take to make CI accept it. Maybe listing the package name would help?

```
    release:
      packages:
      - rxros
```

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -7627,6 +7627,21 @@ repositories:
...
     status: developed
+  rx_ros:
+    doc:
+      type: git
+      url: https://github.com/rosin-project/rx_ros.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/rosin-project/rx_ros-release.git
```
Comment (tfoote) [inline melodic/distribution.yaml]: This repo has no content.

**Review by tfoote — CHANGES_REQUESTED** [review]
This is the failure
```
rosdistro: DEBUG: Skipped "github_manifest_provider()": specified tag "release/melodic/rx_ros/0.1.0-0" is not a git tag
rosdistro: DEBUG: Skip non-bitbucket url "https://github.com/rosin-project/rx_ros-release.git"
rosdistro: DEBUG: Skipped "bitbucket_manifest_provider()": Cannot handle non bitbucket url.
rosdistro: DEBUG: Skipped "git_manifest_provider()": Unable to fetch package.xml: Could not clone repository "https://github.com/rosin-project/rx_ros-release.git" at reference "release/melodic/rx_ros/0.1.0-0"
```

The source repo is valid, though it has an unusual layout and a lot of generated files committed that would be cleaner not to have, such as the generated CMakeLists.txt etc. 

Looking inside the release repository: https://github.com/rosin-project/rx_ros-release there's no evidence that bloom was ever run. I just see an initial commit from @gavanderhoorn 

@henrik7264 as mentioned by Chris if you want to make a full release you need to follow those tutorials. Until you do the release section must be removed. (Note that bloom will open the appropriate PRs for you when you run it.)

**Comment by gavanderhoorn** [comment]
@henrik7264: please contact me about this.

**Comment by gavanderhoorn** [comment]
@henrik7264: I would suggest you close this PR and we submit a correct one at a later time.

**Pull request closed without merging**